test/integration/controlling_test.exs

```diff
@@ -12,12 +12,30 @@ defmodule Oban.Integration.ControllingTest do
     refute_receive {:ok, 1}
     refute_receive {:ok, 2}
 
-    assert :ok = Oban.start_queue(:gamma, 5)
-    assert :ok = Oban.start_queue(:delta, 6)
-    assert :ok = Oban.start_queue(:alpha, 5)
+    assert :ok = Oban.start_queue(queue: :gamma, limit: 5)
+    assert :ok = Oban.start_queue(queue: :delta, limit: 6, local_only: true)
+    assert :ok = Oban.start_queue(queue: :alpha, limit: 5)
 
     assert_receive {:ok, 1}
     assert_receive {:ok, 2}
+
+    assert_raise ArgumentError,
```

Comment:
Taken together these seem rather verbose. Perhaps a helper function and less complete matching could clean it up?

```elixir
assert_invalid_start([queue: nil], "exepcted :queue to be a binary")
assert_invalid_start([queue: :invalid, limit: -1], "expected :limit to be a positive integer")
```

They also seem like good candidates for a separate testing block.

Reply:
> Taken together these seem rather verbose

You're absolutely right, good catch.

> Perhaps a helper function and less complete matching could clean it up?

Yes, I agree, based on your suggestion I introduced `assert_invalid_stop/2` and `assert_invalid_start/2`.

Reply:
> They also seem like good candidates for a separate testing block.

Good idea!

Those helpers should be specific to `Oban.start_queue/2` and `Oban.stop_queue/2`? I mean, something like: `assert_invalid_start_queue_option/1,2` and `assert_invalid_stop_queue_option/1,2` or do you prefer something general to check invalid options? I ask because I'm seeing a lot of the same checks on `worker_test.exs` like:


```elixir
test/oban/worker_test.exs
96:    assert_raise ArgumentError, ~r/unknown option/, fn ->
106:    assert_raise ArgumentError, ~r/expected :queue to be/, fn ->
116:    assert_raise ArgumentError, ~r/expected :max_attempts to be/, fn ->
126:    assert_raise ArgumentError, ~r/expected :priority to be/, fn ->
136:    assert_raise ArgumentError, ~r/expected :tags to be a list/, fn ->
144:    assert_raise ArgumentError, ~r/expected :tags to be a list/, fn ->
154:    assert_raise ArgumentError, ~r/unexpected unique options/, fn ->
162:    assert_raise ArgumentError, ~r/unexpected unique options/, fn ->
170:    assert_raise ArgumentError, ~r/unexpected unique options/, fn ->
178:    assert_raise ArgumentError, ~r/unexpected unique options/, fn ->
186:    assert_raise ArgumentError, ~r/unexpected unique options/, fn ->
```

Or should we also introduce a helper to check the invalid options given to the worker?something like `assert_invalid_worker_option/1,2`, where the message is optional. If this is the case, I think I can introduce those helpers in another PR, to keep things focused. wdyt?